### DISCUSSION TITLE
Amsterdam Area

### DISCUSSION THREAD

**Discussion by Miniterror** [mosesman831/OCI-OcC-Fix#13] [Show and tell]
So i been running the script a while now as my Revolut account (my banking solution) is being declined for PAYG.
Edited the script a little bit as i want to start a instance from a backup bootvolume that contains data.
Has been going on for a while but i think im screwed and wont be able to get it back, anyone else had success recently on the Amsterdam region?

![image](https://github.com/user-attachments/assets/7082c99c-3485-4b19-bf06-0986e7491384)

**Comment by Miniterror** [comment]
@mosesman831 just noticed your PULL request for the new version.
As i still havent succeeded to get me a instance i just changed to the new version.
See below of what im seeying now, seems to be way more informative
![image](https://github.com/user-attachments/assets/dd2064db-7ddf-4f28-8a94-51672e8b90bc)

**Reply by mosesman831** [reply]
It took many many tries to actually spawn an instance as well, there are too many people trying to sign up for a VM as well.
